sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/projectoverview/FailedSamplesRatio.groovy

```diff
@@ -0,0 +1,25 @@
+package life.qbic.business.projectoverview
+/**
+ * <h1>Ratio of failed samples</h1>
+ *
+ * <p>This ratio indicates how many samples of a given total amount have failed</p>
+ *
+ * @since 1.0.0
+ *
+*/
+class FailedSamplesRatio {
+
```

Comment:
I think it would make sense that this class also provides a method which calculates the Failed Sample Ratio, since it's needed for the progressbar down the line

Reply:
It could alternatively be incorporated into the project class since the failed sample calculation is only relevant on a project level

Reply:
Good suggestions, however I dont think we need the failed sample amount for that. The bar should only indicate how far the samples are. The amount of failed samples is therefore not directly needed for that (at least in my imagination for the progress bar calculation). The incorperation of the FailedSampleRatio into the Project class is a nice idea but I need to access this class in the grid to be able to display an integer value as like _1 or 2_.